src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+    private static final int FIRST_UPPERCASE_LETTER_AT_ASCII = 65;
+    private static final int LAST_UPPERCASE_LETTER_AT_ASCII = 90;
+    private static final int FIRST_LOWERCASE_LETTER_AT_ASCII = 97;
+    private static final int LAST_LOWERCASE_LETTER_AT_ASCII = 122;
+    private static final int FIRST_SIGN = 32;
+    private static final int LAST_SIGN = 63;
+    private static final int CARRIAGE_RETURN = 13;
+    private static final int LINE_FEED = 10;
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String data = "";
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            int value = reader.read();
+            while (value != -1) {
+                if ((value >= FIRST_UPPERCASE_LETTER_AT_ASCII
+                        && value <= LAST_UPPERCASE_LETTER_AT_ASCII)
+                        || (value >= FIRST_LOWERCASE_LETTER_AT_ASCII
+                        && value <= LAST_LOWERCASE_LETTER_AT_ASCII)) {
+                    data = data + (char)value;
+                } else if (value >= FIRST_SIGN && value <= LAST_SIGN || value == CARRIAGE_RETURN
+                        || value == LINE_FEED) {
+                    data += " ";
+                }
+                value = reader.read();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file!");
```

Comment:
add original exception object to constructor and file name to message, it's valuable information